src/AzureAuth/Program.cs

```diff
@@ -15,6 +17,10 @@ public class Program
     {
         private static void Main(string[] args)
         {
+            // Use UTF-8 output encoding.
+            // This will impact the NLog Console Target as well as any other Console usage.
+            Console.OutputEncoding = Encoding.UTF8;

```

Comment:
Just curious, where/how is the Console being passed or used after this statement?

Reply:
Also, is it possible to write unit tests for this?

Reply:
Great questions!
1. The `Console` is a built in static class and this property we are changing dictates the encoding the `Console` uses for things like `Console.Write` and `Console.WriteLine`. This impacts everything globally that writes using the console, which for this particular bug - is the JSON output other programs are trying to read. To avoid writing tokens in logs, we only use `Console.Write` when printing the token/json. 
2. This is a change which I would argue doesn't warrant a unit test. It could have an integration test, but we have no good way of writing integration tests at this time, because to fully test this - you _must authenticate_ which we can't do in CI, again, at this time. This is not say unit testing encoding behavior isn't valuable. On the contrary, I _highly_ recommend unit testing encoding related behavior, it can be a very tricky space. However, here - we aren't adding any behavior. The `Console` already knows how to write in utf-8 and I'm not interested in unit testing the C# standard library :)

Reply:
Got it! Thanks Kyle!